Refresh.GameServer/Services/CommandService.cs

```diff
@@ -115,6 +115,16 @@ public void HandleCommand(CommandInvocation command, GameDatabaseContext databas
                 database.SetUnescapeXmlSequences(user, false);
                 break;
             }
+            case "showmoddedcontent":
+            {
+                database.SetShowModdedContent(user, true);
+                break;
+            }
+            case "hidemoddedcontent":
+            {
+                database.SetShowModdedContent(user, false);
+                break;
+            }
```

Comment:
It's important that we use as few characters as possible; typing on console is a pain in the ass.
```suggestion
            case "showmods":
            {
                database.SetShowModdedContent(user, true);
                break;
            }
            case "hidemods":
            {
                database.SetShowModdedContent(user, false);
                break;
            }
```